.github/workflows/build_vagrant.yml

```diff
@@ -22,8 +27,10 @@ jobs:
         brew install --cask virtualbox
 
     - name: Setup Vagrant VM
+      working-directory: ansible
       run: |
-        cd ansible
+        # Temporary fix as the Adopt vagrant vm image is not available on the GitHub Actions executors
```

Comment:
this probably is not needed after change done for getting solaris box file from /userContent